### DISCUSSION TITLE
How possible is the concept of layouts

### DISCUSSION THREAD

**Discussion by ammezie** [unpoly/unpoly#640] [Q&A]
Hi,

Let's say I have two layouts:

App:
```
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Document</title>
</head>
<body up-app>
  <main>
    App layout content
  </main>
</body>
</html>
```

Auth:
```
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Document</title>
</head>
<body up-auth>
  <main>
    Auth layout content
  </main>
</body>
</html>
```

How possible is it to target `[up-app]` from `[up-auth]` (and vice versa) and update its fragment?

**Comment by triskweline** [comment]
You can target the `body` element to swap that element:

```html
<a href="/auth" up-target="body">Sign in</a>
```

Note that we recommend against prefixing your custom attributes with `up-`. That namespace is reserved by Unpoly and may clash with future features. Consider using a prefix custom to your app instead.
